StructureController.cs

```diff
@@ -452,12 +453,20 @@ public string ToString(string selected)
     }
     public override string ToString()
     {
-        string output = "";
+        string output = $"{{\n\"position\": [{this.center_of_mass.x}, {this.center_of_mass.y}, {this.rotator.rotation.z}],\n\"translation\": [{this.translation.x}, {this.translation.y}],\n";
```

Comment:
center_of_mass.x + this.transform.position.x
center_of_mass.y + this.transform.position.y